web/style.css

```diff
@@ -151,6 +152,10 @@ a {
   display: flex;
 }
 
+.button-container .button {
+  cursor: pointer;
+}
+
```

Comment:
i prefer the default cursor for the button as it aligns with how the operating system behaves. imo many websites got it wrong (github included) by using pointer cursor for buttons when it should be reserved for links!